sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -94,11 +94,11 @@ attribution AS (
     id AS user_id,
     fxa_uid,
     created_at AS user_registration_date,
-    mozfun.map.get_key(attribution, "referrer") AS referrer,
-    mozfun.map.get_key(attribution, "utm_medium") AS utm_medium,
-    mozfun.map.get_key(attribution, "utm_source") AS utm_source,
-    mozfun.map.get_key(attribution, "utm_content") AS utm_content,
-    mozfun.map.get_key(attribution, "utm_campaign") AS utm_campaign,
+    attribution.referrer,
```

Comment:
Curious why this no longer requires the get_key construct. Was this introduced in another PR?

Reply:
the underlying data type changed to a struct